INSTALL.rst

```diff
@@ -37,8 +37,7 @@ Requirements
 
 - ``git``
 - ``docker``, ``docker-compose``
-- ``python``, ``pip``, ``pyenv``
-- ``pipenv``
+- ``python``, ``pip``, ``poetry``
```

Comment:
Could you add pyenv to be coherent with next section (Installation) that uses pyenv?

Reply:
Oups it is a mistake.